Resources/Computing/LocalComputingElement.py

```diff
@@ -136,6 +141,14 @@ def _prepareHost(self):
     return S_OK()
 
   def submitJob(self, executableFile, proxy=None, numberOfJobs=1):
+    copyExecutable = os.path.join(self.executableArea, os.path.basename(executableFile))
+    if self.parallelLibrary and executableFile != copyExecutable:
+      # Because we use a parallel library, the executable will become a dependency of the parallel library script
+      # Thus, it has to be defined in a specific area (executableArea) to be found and executed properly
+      # For this reason, we move the executable from its location to executableArea
+      # In python2, shutil.move or os.rename only work when src and dst are on the same FS, which might not be the case
```

Comment:
what happens if src and dst are not on the same FS?

Reply:
I thought the point of `shutil.move` was that it does this correctly? See https://docs.python.org/2.7/library/shutil.html#shutil.move

Reply:
@chrisburr you are right. I should not have written this comment actually, because I want to make a copy instead of moving the file for the moment: the Site Director is in charge of removing the executable file afterwards.

@fstagni as Chris said, `shutil,move` works but `os.rename` returns `OSError: [Errno 18] Invalid cross-device link` if src and dst are not on the same FS.